client/packages/yarn.lock

```diff
@@ -0,0 +1,373 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why this file was added?

Reply:
I made an error, but it has been corrected. Please review the commit 9185d4080308838736851d385e28a5303155f8dc.